frontend/ui/elements/refresh_menu_table.lua

```diff
@@ -1,30 +1,51 @@
+local Device = require("device")
 local UIManager = require("ui/uimanager")
-local util = require("ffi/util")
 local _ = require("gettext")
+local Screen = Device.screen
+local T = require("ffi/util").template
 
-local function custom_1() return G_reader_settings:readSetting("refresh_rate_1") or 12 end
-
-local function custom_2() return G_reader_settings:readSetting("refresh_rate_2") or 22 end
-
-local function custom_3() return G_reader_settings:readSetting("refresh_rate_3") or 99 end
+local function custom(refresh_rate_type)
```

Comment:
nitpick, but for read/understandability, `refresh_rate`**_type** is a bit misleading, `refresh_rate_num` or `custom_refresh_rate_setting_name` is probably clearer.

Reply:
Done :)